poseidon/poseidonMonitor/NorthBoundControllerAbstraction/proxy/bcf/sample_span_fabric_empty.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
should this be `[]` or `{}`?

Reply:
it should be [].  the api call there returns an array of span fabric objects, so in the case that there is nothing to return you get an empty array back, not a null or an empty object.  it looks goofy as a file, but i wanted us to be able to test that case and to be consistent with the way the other tests worked.

Reply:
I only asked since you have the `retval` as `{}` here: https://github.com/CyberReboot/poseidon/pull/653/files#diff-62244599db38059aec5986388ce81867R155

Reply:
oh, i see. the test is the spanArray, and `{}` is the first object in the array. nevermind.

Reply:
yes, the idea is that the mocked api call returns [] to get_span_fabric.  get_span_fabric should only return a single object, not an array, so when the api feeds it [] it should pass off an empty object, or maybe None.